cpu/native/periph/timer.c

```diff
@@ -62,7 +65,14 @@ static struct itimerval itv;
 static unsigned long ts2ticks(struct timespec *tp)
 {
     /* TODO: check for overflow */
-    return((tp->tv_sec * NATIVE_TIMER_SPEED) + (tp->tv_nsec / 1000));
+    if (native_timer_scale < 0) {
```

Comment:
Maybe dedup as much as possible:

```
unsigned long result = (tp->tv_sec * NATIVE_TIMER_SPEED) + (tp->tv_nsec / 1000);
if (native_timer_scale >= 0) {
  result <<= native_timer_scale;
} else {
  result >>= -native_timer_scale;
}
return result;
```

Reply:
The `>=` you just put in to mess with my head ;-)?

Reply:
Done.